.pre-commit-config.yaml

```diff
@@ -15,6 +15,7 @@ repos:
   - id: check-symlinks
   - id: detect-private-key
   - id: end-of-file-fixer
+    exclude: ".test-fixtures\/.*"
```

Comment:
why?:)

Reply:
won't this also be detected with golangci?

also, why not let the fixer do its job and check in the corrected form instead of bypassing it?